repair/repair.cc

```diff
@@ -1192,19 +1199,31 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
         if (repair_tracker().is_aborted(id.uuid)) {
             throw std::runtime_error("aborted by user request");
         }
-
         for (auto shard : boost::irange(unsigned(0), smp::count)) {
             auto f = container().invoke_on(shard, [keyspace, table_ids, id, ranges, hints_batchlog_flushed,
-                    data_centers = options.data_centers, hosts = options.hosts, ignore_nodes] (repair_service& local_repair) mutable {
+                    data_centers = options.data_centers, hosts = options.hosts, ignore_nodes, tables_serialized = std::move(tables_serialized), shard] (repair_service& local_repair) mutable {
+                task_id local_id = task_id::create_random_id();
+                local_repair._task_manager_module->make_task<repair_task_impl>(shard, local_id, keyspace, tables_serialized, "", "", task_id{id.uuid}).get();
```

Comment:
what seastar thread are you counting on?
I'm not sure how that works.
Better use a coroutine lambda and use `co_await` rather than `.get()`